pkgs/development/compilers/ocaml/5.0.nix

```
@@ -0,0 +1,9 @@
+import ./generic.nix {
+  major_version = "5";
+  minor_version = "0";
+  patch_version = "0-beta1";
```

```suggestion
  patch_version = "0~beta1";
```
and remove url below.